src/environments/environment.dev-ng-spree.ts

```diff
@@ -2,8 +2,7 @@ import { DEFAULT_CONFIG } from '../config/default/default';
 
 export const environment = {
   production: false,
-  apiEndpoint: 'http://localhost:3000/',
-  // apiEndpoint: DEFAULT_CONFIG.prodApiEndpoint,
```

Comment:
skip this file change. As new developers, will get stuck with api errors.

Reply:
ok